TestAppTypescript/package.json

```diff
@@ -12,11 +12,11 @@
     "clean": "rimraf artifacts"
   },
   "dependencies": {
-    "appcenter": "file:appcenter-1.8.1.tgz",
-    "appcenter-analytics": "file:appcenter-analytics-1.8.1.tgz",
-    "appcenter-crashes": "file:appcenter-crashes-1.8.1.tgz",
-    "appcenter-link-scripts": "file:appcenter-link-scripts-1.8.1.tgz",
-    "appcenter-push": "file:appcenter-push-1.8.1.tgz",
+    "appcenter": "file:appcenter-1.10.0.tgz",
+    "appcenter-analytics": "file:appcenter-analytics-1.10.0.tgz",
+    "appcenter-crashes": "file:appcenter-crashes-1.10.0.tgz",
+    "appcenter-link-scripts": "file:appcenter-link-scripts-1.10.0.tgz",
+    "appcenter-push": "file:appcenter-push-1.10.0.tgz",
     "react": "16.3.0-alpha.1",
     "react-native": "0.54.4",
```

Comment:
Not absolutely required but would be nice to update `react` and `react-native` to the same version in TestApp (https://github.com/Microsoft/AppCenter-SDK-React-Native/blob/develop/TestApp/package.json#L16-L17)

```
"react": "16.6.0-alpha.8af6728",
"react-native": "0.57.3",
```

Reply:
Can we do that later? Would require to test everything again @dhei

Reply:
Fair enough, let's do it later.